custom_model.py

```diff
@@ -42,7 +43,7 @@ def __init__(
         self.rnn = torch.nn.LSTM(
             input_size=embedding_dim,
             hidden_size=rnn_size,
-            bidirectional=False,
+            bidirectional=bidirectional,
```

Comment:
Generally LSTM bidirectional option is very useful in NLP tasks consider to make it a configurable option